static/js/sefaria/sefaria.js

```diff
@@ -1625,12 +1625,12 @@ Sefaria = extend(Sefaria, {
     // First sort by predefined "top"
     const hebrewTopByCategory = {
       "Tanakh": ["Rashi", "Ibn Ezra", "Ramban", "Sforno"],
-      "Talmud": ["Rashi", "Tosafot"],
+      "Talmud": ["Rashi", 'Rashbam', "Tosafot"],
```

Comment:
I know we don't have linting (yet) but let's try and be consistent between `'` and `"` inside the same array...